app/views/lectures/show.html.erb

```diff
@@ -6,19 +6,22 @@
 </p>
 
 <p>
-  <strong>Date of use:</strong>
-  <%= @lecture.date_of_use %>
-</p>
-
-<p>
   <strong>Question set:</strong>
   <%= @lecture.question_set %>
 </p>
 
-<p>
-  <strong>Start This Lecture:</strong>
-  <%= link_to 'Start', @livelecture_uri %>
-</p>
+
+<% if @lecture.completed %>
```

Comment:
Here's where the biggest change is in this PR. When they click the "view lecture button" they wind up here. If the lecture has already been completed, they can see a page that describes the result of the lecture. If it has not been completed, they join it.